testsuite/test_meeting_processing_file_list.py

```diff
@@ -0,0 +1,146 @@
+from flask import url_for
+from pyquery import PyQuery
+from rq import Queue
+from mrt.models import redis_store, Job, db
+from mrt.meetings.printouts import _process_short_list
+from .factories import RoleUserMeetingFactory
+from .factories import RoleFactory, JobsFactory
+
+
+def test_meeting_manager(app):
+    first_role = RoleFactory(permissions=('manage_meeting',))
+    second_role = RoleFactory(permissions=('view_participant',))
+    first_role_user = RoleUserMeetingFactory(role=first_role)
+    second_role_user = RoleUserMeetingFactory(role=second_role,
+                                              meeting=first_role_user.meeting)
```

Comment:
It's not very clear that both roles use the same meeting. You could initialize the `meeting` variable and then use it throughout the rest of the test. Any of the two options below works:
* `meeting = first_role_user.meeting` (after defining `first_role_user`)
* `meeting = MeetingFactory()` (before anything else)